docs/proposals/timeseries-partitioning-in-compactor.md

```diff
@@ -122,6 +122,37 @@ Now that the planner has produced a compaction plan for the T1-T3 compaction gro
 ![Grouping](/images/proposals/timeseries-partitioning-in-compactor-compact.png)
 T1 - Partition 1-2 was created with hash % 2 == 0, and in order to avoid having duplication information in blocks produced by compaction group 3-8, compaction group 5-8, and compaction group 7-8, we need apply the filter the `%8 == 0` hash, as that’s the hash of the highest partition count.
 
+### Compaction Workflow
+
+1. Compactor initializes Grouper and Planner.
+2. Compactor retrieves block's meta.json and call Grouper to group blocks for compaction.
```

Comment:
Just a FYI for you, if you just do `1.`, then markdown engine will do the number bullet properly, so you don't have to worry about renumbering when you add more bullet point. Example like this 

```
1. hello
1. world
```
which will rendered 

1. hello
1. world


Just FYI, because I see so many people spending efforts making sure their number is right. If you know this already, I'm sorry to repeat it :)

Reply:
I know that. My IDE automatically updates numbers on new item or rearranging.